packages/p2p/src/pages/my-ads/my-ads.jsx

```diff
@@ -17,12 +17,13 @@ const MyAdsState = ({ message }) => (
 );
 
 const MyAds = () => {
-    const { general_store, my_ads_store } = useStores();
+    const { general_store, my_ads_store, my_profile_store } = useStores();
 
     React.useEffect(() => {
         my_ads_store.setIsLoading(true);
         my_ads_store.setShowEditAdForm(false);
         my_ads_store.getAccountStatus();
+        my_profile_store.getPaymentMethodsList();
```

Comment:
oh so this fixes the loading on first time and every time you switch between two tabs?

Reply:
cause shouldn't we try to use the hook useP2PPaymentMethods?

Reply:
nvm i think that'll be too much work, i realised we only use it to get the icons, probs need another card to replace all instances of the store function and replace it with the hook in the future 👍

Reply:
yeah.. it would call even when opening the ad create/edit form.. but its all messed up since the calls are partially converted to hooks at some places.. and writing the hook here will need a lot of changes at other places as well.. so thought will leave it this way for now as we already have a refactoring card for changing these implementations

Reply:
do we need this here? we are already calling this when editing an ad

Reply:
yeah.. we are calling it when calling edit ad as well, but since the response for the api call is slow, the required function is not executed. this issue was not there before since the store value was updated from other places, but now that at some places store call was changed to hooks, the store value doesn't have the updated value on the initial load.

Reply:
pls remove it in edit ad then to avoid duplicate calls

Reply:
yeah.. but what if the user is on the list page, and then someone removes the payment method from BO and then goes to edit page?